packages/styles/settings-tools/_u.screen-reader.scss

```diff
@@ -0,0 +1,13 @@
+// Only display content to screen readers
+
+.sr-only {

```

Comment:
as I said, this utility :
- do not follow the prefix convention (should be `mu-sr-only`)
- I don't think component should depend on utils

we can discuss that